quirks/password-rules.json

```diff
@@ -227,6 +227,9 @@
     },
     "idestination.info": {
         "password-rules": "maxlength: 15;"
+    },
+     "impots.gouv.fr": {
```

Comment:
Can you make this line is indented to match the others?

Reply:
👌👌